plone/content.md

```diff
@@ -121,12 +123,42 @@ $ curl -H 'Accept: application/json' http://localhost:8080/plone
 -	`CORS_ALLOW_HEADERS` - A comma separated list of request headers allowed to be sent by the client, e.g. `X-My-Header`. Defaults to `Accept,Authorization,Content-Type,X-Custom-Header`
 -	`CORS_MAX_AGE` - Indicates how long the results of a preflight request can be cached. Defaults to `3600`
 
+**RELSTORAGE:**
+
+-	`RELSTORAGE_ADAPTER_OPTIONS` - A comma separated list of RelStorage adapter options to set for the plone instance (using [plone.recipe.zope2instance](https://relstorage.readthedocs.io/en/latest/configure-application.html#configuring-plone)). This is required in order to use RelStorage.
+
+All other available environment variables match exactly with RelStorage settings, according to the following table and the [settings specification available on the docs](https://relstorage.readthedocs.io/en/latest/relstorage-options.html).
+
+| Environment variable                      | RelStorage option              | Default value                   |
```

Comment:
Unfortunately the Docker Hub doesn't support markdown tables :disappointed:

Reply:
Table removed :wink: